openvm/src/bus_interaction_handler/bitwise_lookup.rs

```diff
@@ -40,7 +40,8 @@ pub fn handle_bitwise_lookup<T: FieldElement>(
                 ]
             } else {
                 // The result of an XOR can only be a byte and have bits set that are set in either x or y
-                let z_constraint = x.disjunction(y).conjunction(&byte_constraint());
```

Comment:
The bug here was that min/max range constraints were also transferred to z, i.e. if we have x=1..4, y= 3..8,then we would get z = 3..4

Reply:
Wouldn't you get `z = 1..8`? And also the correct mask?